docs/src/gallery/geometries.md

```diff
@@ -103,6 +103,24 @@ hstack(p1, p2)
 ```
 
 
+## [`Geom.candlestick`](@ref)
+
+```@example
+using Gadfly, MarketData
+set_default_plot_size(21cm, 8cm)
+ta = ohlc[1:50]
+plot(
+    x     = timestamp(ta),
+    open  = values(ta.Open),
+    high  = values(ta.High),
+    low   = values(ta.Low),
+    close = values(ta.Close),
+    Geom.candlestick,
+    Scale.color_discrete_manual("green", "red")
+)
+```
```

Comment:
this doc example is identical to the tests.  nice things about tests are that they can double as documentation, but are most useful in that capacity only if they present different examples than in the actual docs.  how about using one of the large historical data sets from MarketData here instead?  i like having real data, instead of synthetic fake data, in docs.

Reply:
ah, I will make the example more complete.
I ran into an issue: there are some unwanted gaps on weekend days, if I set the x-axis as a series of `Date`. Is there a way to remove that gaps?

Reply:
I updated the example in doc.